docs/mobile/events.md

```diff
@@ -63,4 +63,41 @@ The event name MUST be `device.app.lifecycle`.
 | `foreground` | Any time after Activity.onResume() or, if the app has no Activity, Context.startService() has been called when the app was in either the created or background states. |
 <!-- endsemconv -->
 
+## Android ANR
+
+When a client app is not responsive, instrumentation can emit an ANR event. This is an important
+event because it helps developers know when their application is having user-facing performance 
+problems. An periodic application-level watchdog is sometimes used to implement the ANR detector. 
+
+`event.name`: `otel.client.anr`
+
+Event payload:
+
+| Field         | Type     | Description                                                            |
+|---------------|----------|------------------------------------------------------------------------|
+| `threshold`   | double   | Number of unresponsive seconds before the ANR was detected.            | 
+| `stacktrace`  | String   | The stack trace of the main thread at the time the ANR was detected.   |
```

Comment:
One could imagine a much richer data model for the stacktrace. I'm sure this exists in other places.